app/[lang]/not-found.tsx

```diff
@@ -0,0 +1,42 @@
+'use client'
+import { Button } from 'shared'
+import Topbar from 'components/Topbar'
+import Image from 'next/image'
+import { useTranslations, useLang, useLocalizedRoutes } from 'hooks'
+
+export default function NotFound() {
+  const routes = useLocalizedRoutes()
```

Comment:
because he only route here is the home '/' route i don't think you need the useLocalizedRoutes hook